maven-release-plugin/src/site/site.xml

```diff
@@ -34,6 +34,7 @@ under the License.
       <item name="Introduction" href="index.html"/>
       <item name="Goals" href="plugin-info.html"/>
       <item name="Usage" href="usage.html"/>
+      <item name="Upgrade" href="upgrade.html"/>
```

Comment:
Maybe `Upgrade from 2 to 3`, `Migrate from 2 to 3` or simply `Migrate`

Reply:
sorry, I did not see the feedback, I merged without that...